apps/nova/src/components/TutorProfile/Ratings.tsx

```diff
@@ -69,43 +69,50 @@ const Ratings: React.FC<{ id: number; tutorName: string | null }> = ({
     );
 
   return (
-    <div className="flex flex-col my-8 mx-10 gap-10  justify-center">
-      {isEmpty(ratings) ? <NoTutorRatings tutorName={tutorName} /> : null}
-
-      <div className="flex gap-4 flex-wrap justify-center">
-        {ratings.map((rating) => {
-          if (
-            "userId" in rating &&
-            (rating.feedback || rating.userId === user.id)
-          )
-            return (
-              <TutorRatingCard
-                feedback={rating.feedback}
-                imageUrl={rating.image}
-                rating={rating.value}
-                studentId={rating.userId}
-                studentName={rating.name}
-                tutorName={tutorName}
-                owner={rating.userId === user.id}
-              />
-            );
+    <div className="flex flex-col y-8 py-8 mx-10 gap-10  justify-center">
+      {isEmpty(ratings) ? (
+        <NoTutorRatings tutorName={tutorName} />
+      ) : (
+        <div className="grid gap-4 flex-wrap justify-center grid-cols-[repeat(auto-fill,minmax(256px,1fr))]">
+          {ratings.map((rating) => {
+            if (
+              "userId" in rating &&
+              (rating.feedback || rating.userId === user.id)
+            )
+              return (
+                <TutorRatingCard
+                  feedback={rating.feedback}
+                  imageUrl={rating.image}
+                  rating={rating.value}
+                  studentId={rating.userId}
+                  studentName={rating.name}
+                  tutorName={tutorName}
+                  owner={rating.userId === user.id}
+                />
+              );
 
-          if ("ratings" in rating)
-            return (
-              <TutorRatingCardGroup
-                ratings={rating.ratings}
-                tutorName={tutorName}
-                value={rating.value}
-              />
-            );
-        })}
-      </div>
+            if ("ratings" in rating)
+              return (
+                <TutorRatingCardGroup
+                  ratings={rating.ratings}
```

Comment:
You need to only pass the required fileds and not the entir rating info. 

Another issue, you will need to wrap image with `asFullAssetUrl`.

Reply:
@neuodev  but this indicates that we map another time, when I used the type `Omit<Props, "tutorName">` I think it was simpler and Understandable. 
I know we tried to have consistent types, but this will result in less performant code
![image](https://github.com/user-attachments/assets/6b8f93d7-cc57-4043-a4f1-3a3dd275ea7c)

WDYT?